src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -168,14 +210,19 @@ void IDisposable.Dispose()
             // do nothing
         }
 
-        public virtual IEnumerable<SuggestedActionSet> ActionSets
+        public virtual bool HasActionSets
         {
             get
             {
-                return null;
+                return false;
             }
         }
 
+        public virtual Task<IEnumerable<SuggestedActionSet>> GetActionSetsAsync(CancellationToken cancellationToken)
+        {
+            return SpecializedTasks.Default<IEnumerable<SuggestedActionSet>>();
```

Comment:
@heejaechang  I believe SpecializedTask.Default<T>() already does this -

    internal static class SpecializedTasks
    {
....
        public static Task<T> Default<T>()
        {
            return Empty<T>.Default;
        }
...
        private static class Empty<T>
        {
            public static readonly Task<T> Default = Task.FromResult<T>(default(T));
        }
...